src/main/sensors/voltage.c

```diff
@@ -362,3 +363,27 @@ void voltageMeterRead(voltageMeterId_e id, voltageMeter_t *meter)
         voltageMeterReset(meter);
     }
 }
+
+// update voltageStableBits
+// new bit is shifted in every VOLTAGE_STABLE_UPDATE_MS
+// bit 1 is shifted in when voltage did not change more than VOLTAGE_STABLE_MAX_DELTA
+// bit 0 is shifted in othervise
+void voltageStableUpdate(voltageMeter_t* vm)
+{
+    const uint32_t now = millis();
+    if (abs(vm->voltageStablePrevFiltered - vm->displayFiltered) > VOLTAGE_STABLE_MAX_DELTA) {
+        // reset stable voltage reference
+        vm->voltageStablePrevFiltered = vm->displayFiltered;
+        vm->voltageStableBits &= ~BIT(0);  // woltage threshold exceeded in this period
```

Comment:
typo: **w**oltage
i would rather define a enum, ~BIT(0) is not a 100% selfexplenatory term: IS_STABLE in this case.
```c
typedef enum voltageStable_e {
  VOLTAGE_UNSTABLE = 0,
  VOLTAGE_IS_STABLE = ~VOLTAGE_UNSTABLE
} voltageStable_t;
```

or something like this. Or do we already have a global enum type for this?

Reply:
It is just clearing of lowest bit. IMO `&= ~(bits)`  is quite idiomatic in C and used in lot of places of BF.
I'll update the comment